### PR TITLE
Test allo site

### PR DESCRIPTION
**Pull request by eltess**
Mega Apdait

### CODE REVIEW

**Review by grinred — APPROVED** [review]
I guess that all is correct)

**Comment by eltess** [comment]
Mega Pull

**Comment by eltess** [comment]
Mega puk

**Comment by eltess** [comment]
Mega Push

File: src/test/java/AlloTest/PersonSingIn.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+//        tokens.add("@gmail");
+//
+//        for(String l:tokens){
+//            if(name.contains (l)){
+//                System.out.println ("Вы ввели " +l);
+//            }  else {
+//                System.out.println ("Вы должны ввести жемей-почту ");
+//                return;
+//            }
+//        }
```
Comment (StepaniukAndii) [inline src/test/java/AlloTest/PersonSingIn.java:44]: tests should be without comments

Comment (eltess) [reply src/test/java/AlloTest/PersonSingIn.java]: Fixted

**Pull request merged**